app/code/Magento/Swatches/view/adminhtml/web/css/swatches.css

```diff
@@ -149,6 +149,15 @@
     width: 50px;
 }
 
+.col-swatch, [class^=swatch-col] {
+  min-width: 150px;
```

Comment:
Please, use correct indentation for styles

Reply:
Indentation has been corrected.